fabfile.py

```diff
@@ -421,15 +441,16 @@ def snapshot_delete(c, snapshot_id=None):
             raise Exit(result.stderr)
         print(f"Deleted snapshot '{snapshot_id}'.")       
 
-# The list function takes three optional arguments: distros, snapshots and workloads.
+# The list function takes the following optional arguments:
 #    --distros    List the available distributions for launching a testy server.
+#    --instances  Give information about the existing instances.
 #    --snapshots  List the snapshots created from scheduled backups, validation failure,
 #                 and wiredtiger failure.
 #    --workloads  List the workloads available on the specified testy server. The -H
 #                 option is required.
 @task
-def list(c, distros=False, snapshots=False, workloads=False):
-    if not distros and not snapshots and not workloads:
+def list(c, distros=False, instances=False, snapshots=False, workloads=False):
+    if not distros and not instances and not snapshots and not workloads:
```

Comment:
If the optional arguments grow, this check will grow as well and does not look very good. Any idea how to make it better?

Reply:
We can have a variable option_given or similar, set to False at the start, if we enter any if condition set it to true. At the end we can do if option_given and if it is false print Missing arguments otherwise return cleanly. 
- This will only check four if statements if no option is given - the workload is the same basically.

Reply:
Thank you for the input. I decided not to check for anything and add a comment instead, less code and easier to maintain.